scripts/phpcs-new-files

```diff
@@ -10,4 +10,5 @@ git diff-tree \
             -z \
             --name-only \
             --no-commit-id \
+            --diff-filter=ACMRTUXB \
```

Comment:
@arthurbaghdas Is this a change that we still want in this file? Seems unrelated to the ticket.